marlowe-runtime/marlowe-history-cli/Main.hs

```diff
@@ -1,14 +1,27 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GADTs     #-}
 module Main where
 
 import Control.Category ((>>>))
 import Control.Exception (bracket, bracketOnError, throwIO)
-import Data.Foldable (asum, fold)
+import Data.ByteString.Base16 (encodeBase16)
+import Data.Foldable (asum, fold, for_, traverse_)
 import Data.Functor (void)
 import qualified Data.Map as Map
+import qualified Data.Text as T
 import qualified Data.Text.IO as T
 import Data.Void (Void, absurd)
 import GHC.Base (when)
-import Language.Marlowe.Runtime.Core.Api (ContractId, parseContractId, renderContractId)
+import qualified Language.Marlowe.Core.V1.Semantics as V1
+import Language.Marlowe.Pretty (pretty)
+import qualified Language.Marlowe.Protocol.Sync.Client as MarloweSync
+import Language.Marlowe.Protocol.Sync.Codec (codecMarloweSync)
+import qualified Language.Marlowe.Protocol.Sync.Types as Sync
```

Comment:
I'm receiving compile errors due to unused imports:
```
marlowe-history-cli/Main.hs:19:1: error: [-Wunused-imports, -Werror=unused-imports]                                                  
    The qualified import of ‘Language.Marlowe.Protocol.Sync.Types’ is redundant                                                      
      except perhaps to import instances from ‘Language.Marlowe.Protocol.Sync.Types’                                                 
    To import instances alone, use: import Language.Marlowe.Protocol.Sync.Types()                                                    
   |                                                                                                                                 
19 | import qualified Language.Marlowe.Protocol.Sync.Types as Sync                                                                   
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
                                                                                                                                     
marlowe-history-cli/Main.hs:33:1: error: [-Wunused-imports, -Werror=unused-imports]                     
    The import of ‘Peer, PeerHasAgency, PeerRole’                                                                                    
    from module ‘Network.TypedProtocol’ is redundant                                                                                 
   |                                                                                                                                 
33 | import Network.TypedProtocol (Driver (..), Peer (..), PeerHasAgency (..), PeerRole (..), runPeerWithDriver)                     
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                     
```

Reply:
Odd, the CI should be failing in that case...

Reply:
I did a fresh clone, `nix develop`, and `cabal build all`.

(This is not blocking my review: I fixed the problem locally, so I can test the executables.)